docs/getting-started/integrations-cdn/angular.md

```diff
@@ -18,6 +18,8 @@ order: 30
 
 Angular is a TypeScript-based, open-source, single-page web application framework. The CKEditor 5 component for Angular supports integrating different editor types.
 
+This guide assumes you already have a Angular project. To create such a project, you can use Angular CLI. Refer to the [Angular documentation](https://angular.io/cli) to learn more.
```

Comment:
```suggestion
This guide assumes you already have an Angular project. To create such a project, you can use Angular CLI. Refer to the [Angular documentation](https://angular.io/cli) to learn more.
```

Reply:
I don't know if it's a good place for this paragraph, but I would keep it in some form. Some people were confused when we didn't explicitly say: "Hey, you need a project first."

Reply:
I would keep it, too. I believe the place is as good as any before describing the procedure.